src/main/java/Main.java

```diff
@@ -0,0 +1,54 @@
+import com.solvd.db.dao.mysql.AccountDAO;
+import com.solvd.db.model.*;
+import com.solvd.db.service.CustomerService;
+import com.solvd.db.service.PersonService;
+
+import java.sql.*;
+import java.text.ParseException;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
+public class Main {
+    public static void main(String[] args) throws InterruptedException, SQLException, ParseException {
```

Comment:
Better to handle exceptions than to have main throw

Reply:
they have been removed, they are handled in other methods